build.gradle

```diff
@@ -22,6 +22,8 @@ allprojects {
 
     repositories {
         jcenter()
+        mavenLocal()
```

Comment:
This is temporary.  Have built com.mesosphere:marathon-client:0.3.1-SNAPSHOT locally.